plugins/nodes/src/js/components/NodesTable.tsx

```diff
@@ -0,0 +1,201 @@
+import * as React from "react";
+import { Table, Column } from "@dcos/ui-kit";
+
+import NodesList from "#SRC/js/structs/NodesList";
+import Node from "#SRC/js/structs/Node";
+import { IWidthArgs as WidthArgs } from "#PLUGINS/nodes/src/js/types/IWidthArgs";
+
+import { SortableColumnHeader } from "ui-kit-stage/SortableColumnHeader";
+import { SortDirection } from "../types/SortDirection";
+
+import * as NodesTableHostnameColumn from "../columns/NodesTableHostnameColumn";
+import * as NodesTableRegionColumn from "../columns/NodesTableRegionColumn";
+import * as NodesTableZoneColumn from "../columns/NodesTableZoneColumn";
+import * as NodesTableHealthColumn from "../columns/NodesTableHealthColumn";
+import * as NodesTableTasksColumn from "../columns/NodesTableTasksColumn";
+import * as NodesTableCPUBarColumn from "../columns/NodesTableCPUBarColumn";
+import * as NodesTableCPUColumn from "../columns/NodesTableCPUColumn";
+import * as NodesTableMemBarColumn from "../columns/NodesTableMemBarColumn";
+import * as NodesTableMemColumn from "../columns/NodesTableMemColumn";
+import * as NodesTableDiskBarColumn from "../columns/NodesTableDiskBarColumn";
+import * as NodesTableDiskColumn from "../columns/NodesTableDiskColumn";
+import * as NodesTableGPUBarColumn from "../columns/NodesTableGPUBarColumn";
+import * as NodesTableGPUColumn from "../columns/NodesTableGPUColumn";
+
+interface NodesTableColumn {
+  displayName: string;
```

Comment:
this is not a string (will become important below). Also please consider renaming it, e.g.: `headerContent?: React.ReactNode;`